VSRAD.Package/DebugVisualizer/SliceVisualizer/TypedSliceWatchView.cs

```diff
@@ -124,14 +125,14 @@ private static void GetMinMaxValues(SliceWatchView view, VariableType type, out
                             byte[] bytes = BitConverter.GetBytes(view[row, col]);
                             float firstHalf = Half.ToFloat(BitConverter.ToUInt16(bytes, 0));
                             float secondHalf = Half.ToFloat(BitConverter.ToUInt16(bytes, 2));
-                            if (!float.IsNaN(firstHalf))
+                            if (!float.IsNaN(firstHalf) && !float.IsInfinity(firstHalf))
                             {
                                 if (firstHalf < fmin)
                                     fmin = firstHalf;
                                 if (firstHalf > fmax)
                                     fmax = firstHalf;
                             }
-                            if (!float.IsNaN(secondHalf))
+                            if (!float.IsNaN(secondHalf) && !float.IsInfinity(firstHalf))
```

Comment:
should probably be `IsInfinity(secondHalf)`

Reply:
Thanks, fixed in 43da773